core/src/main/java/org/mobilitydata/gtfsvalidator/validator/ValidatorLoader.java

```diff
@@ -141,12 +141,18 @@ public ValidatorLoader(ImmutableList<String> validatorPackages) throws Validator
       throws ValidatorLoaderException {
     Constructor<T> constructor = chooseConstructor(validatorClass);
 
+    // Indicates that the full GtfsFeedContainer needs to be injected.
```

Comment:
Could you explain why this change was needed please?

Reply:
Please see usage example in TranslationsValidator.java. translations.table_name may reference any table, so we locate it by name in GtfsFeedContainer.

```
  @Inject
  TranslationsValidator(
      GtfsTranslationTableContainer translationTable, GtfsFeedContainer feedContainer) {
    this.translationTable = translationTable;
    this.feedContainer = feedContainer;
  }
  ...

    Optional<GtfsTableContainer<?>> table = feedContainer.getTable(translation.tableName());
    if (!table.isPresent()) {
      noticeContainer.addValidationNotice(new TranslationUnknownTableNameNotice(translation));
      return;
    }

```